src/routes/Edit/VideoSplitter.tsx

```diff
@@ -367,6 +403,41 @@ const VideoSplitter = ({
     );
   }
 
+  const convertTags = (rank: number) => {
```

Comment:
Lets go ahead and revert to the old tag system for now and move these features to a new branch until this functionality is more mature.

Reply:
I have separated the drag and drop feature for now. We will implement it later. It is now working fine, and implements the drop down suggestion feature mainly.
I left the old code there, so if you stick to this decision, we will revert back quickly.